wolfcrypt/src/random.c

```diff
@@ -2026,6 +2030,12 @@ int wc_FreeRng(WC_RNG* rng)
         ret = WC_HW_E;
 #endif
 
+#ifndef USE_WINDOWS_API
+    if(rng->seed.fd != -1) {
+        close(rng->seed.fd);
+    }
```

Comment:
Should this also reset `rng->seed.fd = -1;`?

Reply:
Great catch, it should indeed.  Fixed.